cg_lims/EPPs/udf/set/set_barcode.py

```diff
@@ -74,24 +90,27 @@ def get_barcode_set_udf(
 @click.command()
 @options.artifact_udf(help="The name of the barcode udf.")
 @options.input()
+@options.measurement()
 @options.container_type()
 @click.pass_context
 def assign_barcode(
     ctx: click.Context,
     artifact_udf: str,
     input: bool,
+    measurement: bool,
```

Comment:
Detta sätter default till False, om inte annat har angets när funktionen kallas.
```suggestion
    measurement: bool = False,
```

Reply:
Den sätts default False på options, men håller med - denna lösning är bättre :)